scripts/check.tokens.erc20.ts

```diff
@@ -0,0 +1,43 @@
+import type { Erc20Contract } from '$eth/types/erc20';
+import {
+	ERC20_CONTRACTS_PRODUCTION,
+	ERC20_CONTRACTS_SEPOLIA,
+	ERC20_TWIN_TOKENS_MAINNET,
+	ERC20_TWIN_TOKENS_SEPOLIA
+} from '../src/frontend/src/env/tokens.erc20.env';
+
+const getAddresses = (contracts: Erc20Contract[]) =>
```

Comment:
NTH: Given that's a TS script, can you express the return types of the function?

For example:

```
const getAddresses = (...): boolean => 
```

Reply:
done